packages/foundation/src/next/composed.tsx

```diff
@@ -145,7 +145,14 @@ export function composed<
       _defaultStyles: displayName ? finalStyles : styles
     } as TViewProps & IDefaultSlotProps<any>;
 
-    return view ? view(viewProps) : null;
+    let Slots: ISlots<Required<TComponentSlots>>;
+    if (options.slots) {
+      Slots = typeof options.slots === 'function' ? getSlots(viewProps, options.slots(viewProps)) : getSlots(viewProps, options.slots);
+    } else {
+      Slots = {} as ISlots<Required<TComponentSlots>>;
```

Comment:
Might be worth throwing a dev mode only error here for now. I can't think of a valid reason for this code to ever be hit if everything has been set up correctly.

Reply:
How can you specify to throw an error in dev mode only?

Reply:
Is it just `throw new Error( ... )`

Reply:
You can check `process.env.NODE_ENV`, but it seems most of the other instances of `throw new Error` in our codebase don't check it. Either way is fine for now, I think.

Reply:
I'm also wondering, should it be an error or a warning? What if the user is creating a non-slottable component?

Reply:
If they are making a non-slottable component, then the component's Slots interface will be empty. We can depend on type safety to prevent users from actually trying to use Slots that don't exist via typing. That means they can't trigger a thrown error without completely violating typing.